test/unit/plugins/validate-semantic/oas3/refs.js

```diff
@@ -206,4 +206,82 @@ describe("validation plugin - semantic - oas3 refs", () => {
       return expectNoErrorsOrWarnings(spec)
     })
   })
+
+  describe("$refs for requestbody schemas must reference a schema by position", () => {
+    it("should return an error when a requestBody schema incorrectly references a local component schema", () => {
+      const spec = {
+        openapi: "3.0.0",
+        paths: {
+          "/foo": {
+            post: {
+              requestBody: {
+                content: {
+                  "application/json": {
+                    schema: {
+                      $ref: "#/components/requestbodies/Foo"
```

Comment:
@ralphdoe, I believe the two extra errors you're seeing are due to the fact that `#/components/requestbodies/Foo` doesn't actually exist in your spec object — consider changing this to `#/components/requestBodies/Foo` and L230 below from `schemas` to `requestBodies`!

Reply:
Done!

Reply:
Done!